### PR TITLE
Pioneer 20c glider telem ingestions

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: CP15MOAS-GL339/CP15MOAS-GL339_D00001_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL339/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL339-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL339/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL339-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL339/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL339-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL339/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL339-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL339/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL339-05-PARADM000,telemetered,Available,
```
Comment (cooleyky) [inline CP15MOAS-GL339/CP15MOAS-GL339_D00001_ingest.csv:6]: looks good

File: CP15MOAS-GL380/CP15MOAS-GL380_D00001_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL380/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL380-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL380/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL380-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL380/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL380-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL380/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL380-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL380/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL380-05-PARADM000,telemetered,Available,
```
Comment (cooleyky) [inline CP15MOAS-GL380/CP15MOAS-GL380_D00001_ingest.csv:6]: looks good

File: CP15MOAS-GL559/CP15MOAS-GL559_D00001_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL559/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL559-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL559/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL559-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL559/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL559-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL559/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL559-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL559/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-GL559-05-PARADM000,telemetered,Available,
```
Comment (cooleyky) [inline CP15MOAS-GL559/CP15MOAS-GL559_D00001_ingest.csv:6]: looks good

File: CP15MOAS-PG564/CP15MOAS-PG564_D00002_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-PG564/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-PG564-01-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-PG564/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-PG564-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-PG564/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-PG564-03-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_o.flort_o_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-PG564/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-PG564-04-FLORTO000,telemetered,Available,
+mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-PG564/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-PG564-05-NUTNRM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-PG564/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-PG564-06-PARADM000,telemetered,Available,
```
Comment (cooleyky) [inline CP15MOAS-PG564/CP15MOAS-PG564_D00002_ingest.csv:8]: looks good

**Review by cooleyky — APPROVED** [review]
looks good, and all data in raw data repo

File: CP15MOAS-GL339/CP15MOAS-GL339_D00001_ingest.csv (hunk 1 above)
Comment (rtravis-whoi) [inline CP15MOAS-GL339/CP15MOAS-GL339_D00001_ingest.csv:6]: looks good - data on raw data repo

File: CP15MOAS-GL380/CP15MOAS-GL380_D00001_ingest.csv (hunk 2 above)
Comment (rtravis-whoi) [inline CP15MOAS-GL380/CP15MOAS-GL380_D00001_ingest.csv:6]: looks good - data on raw data repo

File: CP15MOAS-GL559/CP15MOAS-GL559_D00001_ingest.csv (hunk 3 above)
Comment (rtravis-whoi) [inline CP15MOAS-GL559/CP15MOAS-GL559_D00001_ingest.csv:6]: looks good - data on raw data repo

File: CP15MOAS-PG564/CP15MOAS-PG564_D00002_ingest.csv (hunk 4 above)
Comment (rtravis-whoi) [inline CP15MOAS-PG564/CP15MOAS-PG564_D00002_ingest.csv:8]: looks good - data on raw data repo

**Review by rtravis-whoi — APPROVED** [review]

**Pull request merged**